### ISSUE TITLE
When overlay is opened with arrow keys, input looses the focus and user cannot write

### ISSUE THREAD

**Issue by manolo** [vaadin/vaadin-date-picker#556]
It should be aligned with other elements, e.g. in combo-box, when the item selector is opened the user still can write in the input, ie input never looses the focus

In date-picker the behaviour it's a bit confusing, opening with any key the focus remains in input, but with arrow keys focus goes to the overlay.

The issue might be in the focus loop, when opened with arrow up/down keys calendar takes focus, then you can loop to buttons or input. Probably this intermediate focus should be skip, keeping it in input, and focus loops just input -> buttons

**Comment by jouni** [comment]
This is actually by design. I’m not 100% sure it’s the best possible design, so I’m okay if we want to reopen this decision. But I would not want to do any hasty changes.

**Comment by manolo** [comment]
> But I would not want to do any hasty changes.

Yes that is the motivation of this issue, discuss whether date-picker should behave different to combo-box.  accessibility-wise  I think focus should remain in text-field

> conflict with left and right arrow keys
